src/renderer/components/kube-object/kube-object-list-layout.tsx

```diff
@@ -17,6 +17,10 @@ export interface KubeObjectListLayoutProps extends ItemListLayoutProps {
 
 @observer
 export class KubeObjectListLayout extends React.Component<KubeObjectListLayoutProps> {
+  static defaultProps: Partial<KubeObjectListLayoutProps> = {
```

Comment:
You cannot use it like this (not sure exactly why though), but it leads to that all props becomes `optional`.
For example comment out required `store={...}` in some place of use and you'll see no complains from TS.

Workaround would be as done in other places:

```ts
const defaultProps: Partial<Props> = {
  // define optionals
};

export class ComponentName extends React.Component<Props> {
  static defaultProps = defaultProps as object;
```

Reply:
Ah fair thanks for the catch. I think we still do this in some places but that should be a different PR.